matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/create/CreateRoomTask.kt

```diff
@@ -44,9 +44,14 @@ import org.matrix.android.sdk.internal.util.awaitTransaction
 import org.matrix.android.sdk.internal.util.time.Clock
 import java.util.concurrent.TimeUnit
 import javax.inject.Inject
+import javax.inject.Qualifier
 
 internal interface CreateRoomTask : Task<CreateRoomParams, String>
 
+@Qualifier
+@Retention(AnnotationRetention.BINARY)
+annotation class Default
+
```

Comment:
I added a second implementation of the interface, so I added an annotation for each implementation to be able to choose which one I want to inject, is it OK?